testplan/common/remote/remote_resource.py

```diff
@@ -625,7 +625,7 @@ def _remote_copy_path(self, path):
         suitable for use in a copy command such as `scp`.
         """
         return "{user}@{host}:{path}".format(
-            user=self._user, host=self.cfg.remote_host, path=path
+            user=self._user, host=self.ssh_cfg["host"], path=path
```

Comment:
needs to update 
https://github.com/morganstanley/testplan/blob/eddfbd8f186071dd3c3d5a9878be75d52de35962/testplan/common/remote/remote_resource.py#L535

https://github.com/morganstanley/testplan/blob/eddfbd8f186071dd3c3d5a9878be75d52de35962/testplan/common/remote/remote_resource.py#L538

https://github.com/morganstanley/testplan/blob/eddfbd8f186071dd3c3d5a9878be75d52de35962/testplan/common/remote/remote_resource.py#L556

self.cfg.remote_host -> self.ssh_cfg["host"]